Simplenote/src/main/java/com/automattic/simplenote/di/AppModule.kt

```diff
@@ -0,0 +1,16 @@
+package com.automattic.simplenote.di
+
+import android.content.Context
+import com.automattic.simplenote.Simplenote
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+
+@Module
+@InstallIn(SingletonComponent::class)
+class AppModule {
```

Comment:
(:pick:) Change the app module to `object` instead of `class`:
```suggestion
object AppModule {
```